at_client/test/verb_builder_test.dart

```diff
@@ -0,0 +1,140 @@
+import 'package:at_client/at_client.dart';
+import 'package:at_commons/at_builders.dart';
+import 'package:at_commons/at_commons.dart';
+import 'package:test/test.dart';
+import 'package:at_client/src/client/verb_builder_manager.dart';
+
+import 'samples/test_util.dart';
+
+void main() {
+  group('A group of tests to verify getSecondary Instance', () {
+    setUp(() async {
+      final atSign = '@alice';
+      await AtClientManager.getInstance()
+          .setCurrentAtSign(atSign, 'wavi', TestUtil.getMuraliPreference());
```

Comment:
getMuraliPreference should probably be renamed :-)

Reply:
Renamed the preference.